### PR TITLE
Added User Guide

### PR DESCRIPTION
**Pull request by blackstardlb**
Added a User Guide and made some changes to the readme.

### CODE REVIEW

File: docs/USER_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
+# Flint IDE User Guide  
+This is the MPS Flint IDE user guide. The Guide covers the following topics.
+- [Creating a Flint project](#creating-a-flint-project)
+- [Import existing flint model](#optional-import-existing-flint-model)
+- [Creating a Flint project](#creating-a-flint-project)
+- [Editing a FlintModel](#editing-a-flintModel)
+- [Creating a FlintRunner](#creating-a-flintRunner)
```
Comment (n9iels) [inline docs/USER_GUIDE.md:7]: The `Creating a Flint project` is listed twice

**Comment by n9iels** [comment]
A good addition to this manual might be the CTRL + B shortcut to jump directly to a definition of the fact from within a act form example. Something I did automatically  since it its a jetbrains product :smile:

File: docs/USER_GUIDE.md (hunk 1 above)
Comment (blackstardlb) [reply docs/USER_GUIDE.md:7]: On is creating flint project and one is creating flintrunner

Comment (n9iels) [reply docs/USER_GUIDE.md:7]: It is listed on both line 3 and 5

**Pull request merged**